packages/printbox/printbox.v0.3/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+version: "0.3"
+authors: ["Simon Cruanes" "Guillaume Bury"]
+maintainer: "simon.cruanes.2007@m4x.org"
+synopsis: "Allows to print nested boxes, lists, arrays, tables in several formats"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```
Dune should not be tagged as a build dependency (see #14266) and your `dune-project` file states that this package requires at least dune 1.1